src/main/java/bio/terra/service/tabulardata/google/BigQueryPdao.java

```diff
@@ -179,29 +179,37 @@ public void createDataset(Dataset dataset) throws InterruptedException {
    * Construct the live view sql.
    *
    * @param activeTransaction If not null, include values from this transaction. Note, if this has a
-   *     non-null values then you MUST set the transactId query parameter. This means that this
-   *     value can not be non-bull when building a view
+   *     non-null value then you MUST set the transactId query parameter. This means that this value
+   *     can not be non-bull when building a view
+   * @param creationStart If not null, only return values from transactions that were committed
```

Comment:
The name of this term is confusing to me. I understand that the source of this value is the creation start, but as far as this method is concerned, its meaning is actually `filterBefore` or something like that. If the caller generated the value in some other way, it might not be a creation start time.

To think about it another way, it's either a token whose meaning is opaque to the caller (e.g. its source can only be from a transaction operation), or it's a time value whose source might be creation but could also be some other event.

Reply:
That's a great point.  I prefer your suggestion of making it more generic.  I'll update